device-types/Lenovo/system-x3650-M5.yaml

```diff
@@ -0,0 +1,60 @@
+---
+manufacturer: Lenovo
+model: System x3650 M5
+slug: lenovo-system-x3650-m5
+part_number: 8871
+u_height: 2
+is_full_depth: true
+comments: '[Lenovo System x3650 M5 product guide](https://lenovopress.com/tips1194.pdf)'
+power-ports:
```

Comment:
As these are hot swappable PSUs they need to be modeled as `module-bays` and then add the require PSUs as separate modules if they don't already exist